app/controllers/people_controller.rb

```diff
@@ -0,0 +1,18 @@
+class PeopleController < ApplicationController
+  def index
+    @people = Person.order(id: :desc)
+  end
```

Comment:
I'm not sure but I believe using .all.sort might be slightly more efficient and save some run time.  Here you are technically sorting twice - ascending and then descending.  I could be wrong.